src/app/(authenticated)/settings/page.tsx

```
@@ -110,6 +115,17 @@ export default function UserSettings() {
     );
   }
 
+  // Show error toast if there's an error
+  useEffect(() => {
```

ESLint

[error] React Hook "useEffect" is called conditionally. React Hooks must be called in the exact same order in every component render.

(react-hooks/rules-of-hooks)